chats/apps/api/v1/queues/viewsets.py

```diff
@@ -12,7 +12,7 @@
 
 
 class QueueViewset(ModelViewSet):
-    queryset = Queue.objects.all()
+    queryset = Queue.objects.exclude(is_deleted=True)
```

Comment:
Don't need to exclude here, it'll be possible to return the rooms from deleted queues. Better to add a new filter(is_deleted) with the default excluding the deleted from the query